### PR TITLE
Combat Defib - Cell Change

### PR DESCRIPTION
**Pull request by Kiyahitayika**
## What Does This PR Do
Exchanges the infinite cell in the Nukie Combat Defib with the regular cell that all defibs have (10k maxcharge)
Line 244:
	`cell = new /obj/item/stock_parts/cell/infinite(src)`
to
	`cell = new(src)`

I also went through and changed various 1/0 variables to TRUE/FALSE variables so AA wouldn't beat me.

## Why It's Good For The Game
Slight nerf to Nukie Combat Defib - discussed in Discord.

## Changelog
:cl:
tweak: The cell in the Nukie Combat Defib now has 1/3 the maximum charge.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/defib.dm [hunk 1]
```diff
@@ -113,21 +113,21 @@
...
+		safety = TRUE
 		to_chat(user, "<span class='notice'>You silently enable [src]'s safety protocols with the card.")
 
 /obj/item/defibrillator/emp_act(severity)
 	if(cell)
 		deductcharge(1000 / severity)
 	if(safety)
-		safety = 0
+		safety = FALSE
 		src.visible_message("<span class='notice'>[src] beeps: Safety protocols disabled!</span>")
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/defib.dm:127]:
```suggestion
		visible_message("<span class='notice'>[src] beeps: Safety protocols disabled!</span>")
```

File: code/game/objects/items/weapons/defib.dm [hunk 2]
```diff
@@ -113,21 +113,21 @@
...
 		playsound(get_turf(src), 'sound/machines/defib_saftyoff.ogg', 50, 0)
 	else
-		safety = 1
+		safety = TRUE
 		src.visible_message("<span class='notice'>[src] beeps: Safety protocols enabled!</span>")
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/defib.dm:131]:
```suggestion
		visible_message("<span class='notice'>[src] beeps: Safety protocols enabled!</span>")
```

File: code/game/objects/items/weapons/defib.dm [hunk 3]
```diff
@@ -339,7 +339,7 @@
 		return
 	else
 		if(user.a_intent == INTENT_HARM && !defib.safety)
-			busy = 1
+			busy =TRUE
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/defib.dm:342]:
```suggestion
			busy = TRUE
```

File: code/game/objects/items/weapons/defib.dm [hunk 4]
```diff
@@ -351,13 +351,13 @@
...
-			cooldown = 1
-			busy = 0
+			cooldown = TRUE
+			busy = FALSE
 			update_icon()
 			defib.cooldowncheck(user)
 			return
 		user.visible_message("<span class='warning'>[user] begins to place [src] on [M.name]'s chest.</span>", "<span class='warning'>You begin to place [src] on [M.name]'s chest.</span>")
-		busy = 1
+		busy =TRUE
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/defib.dm:360]:
```suggestion
		busy = TRUE
```

File: code/game/objects/items/weapons/defib.dm [hunk 5]
```diff
@@ -490,7 +490,7 @@
 		return
 	else
 		if(user.a_intent == INTENT_HARM  && !safety)
-			busy = 1
+			busy =TRUE
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/defib.dm:493]:
```suggestion
			busy = TRUE
```

File: code/game/objects/items/weapons/defib.dm [hunk 6]
```diff
@@ -490,7 +490,7 @@
 		return
 	else
 		if(user.a_intent == INTENT_HARM  && !safety)
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/defib.dm:492]:
```suggestion
		if(user.a_intent == INTENT_HARM && !safety)
```

File: code/game/objects/items/weapons/defib.dm [hunk 7]
```diff
@@ -504,15 +504,15 @@
...
+			busy = FALSE
 			update_icon()
 			spawn(50)
-				cooldown = 0
+				cooldown = FALSE
 				update_icon()
 			return
 		user.visible_message("<span class='warning'>[user] begins to place [src] on [M.name]'s chest.</span>", "<span class='warning'>You begin to place [src] on [M.name]'s chest.</span>")
-		busy = 1
+		busy =TRUE
```
Comment (AffectedArc07) [inline code/game/objects/items/weapons/defib.dm:515]:
```suggestion
		busy = TRUE
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/defib.dm [hunk 8]
```diff
@@ -235,13 +235,13 @@
...
-	combat = 1
-	safety = 0
+	combat = TRUE
+	safety = FALSE
 
 /obj/item/defibrillator/compact/combat/loaded/New()
 	..()
 	paddles = make_paddles()
-	cell = new /obj/item/stock_parts/cell/infinite(src)
+	cell = new(src)
```
Comment (Crazylemon64) [inline code/game/objects/items/weapons/defib.dm:244]: This appears to be the functional change in this PR

Comment (Kiyahitayika) [reply code/game/objects/items/weapons/defib.dm:244]: Yes.
> Exchanges the infinite cell in the Nukie Combat Defib with the regular cell that all defibs have (10k maxcharge)

**Review by variableundefined — APPROVED** [review]

**Comment by variableundefined** [comment]
Approved for design

**Review by Kyep — APPROVED** [review]

**Pull request merged**